salt/utils/http.py

```diff
@@ -120,6 +120,7 @@ def query(url,
           stream=False,
           handle=False,
           agent=USERAGENT,
+          hide_fields=None,
```

Comment:
This is breaking (almost) all salt.utils.http.query() calls where ``hide_fields`` isn't overriden. fixed by #28294